### ISSUE TITLE
Incorrect work of the patcher

### ISSUE THREAD

**Issue by Russianstory** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#1012]
After installing the latest patcher 2.15b and configuring it. Computer opponents, although they are gaining an army of 600 units, do not attack at all. What did I do wrong? What parameters do I need to enable so that the enemy:
1) recruited an army quickly
2) Gained more each time
3) Attacked often

**Comment by gynt** [comment]
Please make sure your issue title reflects the content of your issue. I fixed it for you.

Can you share your ucp.cfg file and a .sav file of this situation?

**Comment by Russianstory** [comment]
Yes, I agree about the title. Let it be the way it's supposed to be! =)

About the file yes of course here it is

**Comment by Russianstory** [comment]
https://drive.google.com/file/d/1NTmBBvwc5f7auHbO3Dk32JHvHLUPWtO7/view?usp=share_link

**Comment by Russianstory** [comment]
Can't you download the archive itself?

**Comment by Russianstory** [comment]
pls try to download this one https://dropmefiles.com/cwu6p

**Comment by Russianstory** [comment]
And?
